packages/mgt-components/src/components/mgt-file-list/mgt-file-upload/mgt-file-upload.ts

```diff
@@ -339,6 +339,17 @@ export class MgtFileUpload extends MgtBaseComponent {
   constructor() {
     super();
     this.filesToUpload = [];
+    this.addEventListener('__uploadfailed', this.focusOnUpload)
+    this.addEventListener('__uploadsuccess', this.focusOnUpload)
+  }
+
+  focusOnUpload() {
+    const uploadDom = this.renderRoot.querySelector('mgt-file[part="upload"]')
```

Comment:
```suggestion
    const uploadDom: HTMLElement | null = (this.renderRoot as ShadowRoot).querySelector('mgt-file[part="upload"]');
```

Reply:
Why are you casting `this.renderRoot` as `ShadowRoot`?

Reply:
Yeah, that's unnecessary. Edited the suggestion